tri_again/test_collada.py

```diff
@@ -34,9 +34,9 @@ def test_example():
 
 def test_noncontiguous_polyline_vertices():
     polyline = Polyline(
-        np.flipud(np.array([[5.5, 0.5, 0.0], [5.5, 0.75, 0.0], [5.5, 0.5, -0.5]])),
-        is_closed=True,
+        np.array([[5.5, 0.5, 0.0], [5.5, 0.75, 0.0], [5.5, 0.5, -0.5]]), is_closed=True
     )
+    polyline.v = np.flipud(polyline.v)
```

Comment:
Polyline is intended to be immutable so this is a little odd. Though I wonder if that is why the problem has gone away – that we have in a way fixed the issue, such that the Polyline constructor now returns always contiguous vertices.

If that's the case, perhaps the thing to do is require the polliwog version with immutability (i.e. 3.0.0a0 or later) and write a new test in polliwog that asserts that `polyline.v` is contiguous even when the input verts are not.

Reply:
OK, I've removed this test, and added a test in Polliwog here: https://github.com/lace/polliwog/pull/350